meshtastic/admin.proto

```diff
@@ -376,6 +382,21 @@ message AdminMessage {
      * Tell the node to reset the nodedb.
      */
     int32 nodedb_reset = 100;
+
```

Comment:
I would put them after set_time_only since 64 and higher are the management commands